### ISSUE TITLE
tldr-node-client doesnt recognise my operating system (fresh install on Windows).

### ISSUE THREAD

**Issue by sarturodev** [tldr-pages/tldr-node-client#368]
## Expected behavior
tldr-node-client should recognise my operating system and display the page associated with the platform (windows). 
## Actual behavior
tldr-node-client doesnt not recognize my o.s (window 10). This happens after a fresh install.
When i try to write for example: `tldr ipconfig` , it prompts: "Command ipconfig does not exist for the host platform. Displaying the page from windows platform .."
## Log, debug output
[tldr-node-client installation debug-log](https://pastebin.com/51KpB6QM)
## Environment
- Operating system - macOS/Linux/Windows: Windows 10 Pro Ver. 20H2 
- Node.js version: 16.13.0
- tldr-node-client version: 3.3.7

**Comment by sarturodev** [comment]
I've submitted a pull request (#369) that solves this bug.
